app/src/main/res/values/themes.xml

```diff
@@ -52,6 +52,12 @@
     <item name="android:textColorPrimary">@color/component_color_shared_primary_dark_text_color</item>
   </style>
 
+  <style name="DeprecationAlertDialogTheme" parent="OppiaAlertDialogTheme">
+    <item name="android:paddingBottom">3dp</item>
```

Comment:
Should be 4, per android [best practices](https://github.com/oppia/oppia-android/wiki/Working-on-UI#paddingmarginview-dimensions)